core/queryalgebra/evaluation/src/main/java/org/eclipse/rdf4j/query/algebra/evaluation/util/ValueComparator.java

```diff
@@ -87,6 +89,14 @@ public int compare(Value o1, Value o2) {
 		return compareTriples((Triple) o1, (Triple) o2);
 	}
 
+	public void setStrict(boolean flag) {
+		this.strict = flag;
+	}
+
+	public boolean getStrict() {
```

Comment:
This is nitpicking a bit, but the convention is to name methods that return a boolean value `is...` rather than `get...`. So `isStrict()`.

Reply:
this is done.